### PR TITLE
Correct typos in vignette dispatch

### PR DESCRIPTION
**Pull request by ramiromagno**

### CODE REVIEW

**Review by hadley — COMMENTED** [review]
Thanks for working on this! Two small comments below.

File: vignettes/dispatch.Rmd [hunk 1]
```diff
@@ -107,12 +107,12 @@ In this case we really do want to silently ignore unknown arguments because they
 There's unfortunately no easy way to avoid this problem without relying on fairly esoteric technology (as done by `rlang::check_dots_used()`).
 
 ```{r}
-simple_print(list(1, 2, "x"), diggits = 3)
```
[the remark this answers is not in the document]
Comment (ramiromagno) [reply vignettes/dispatch.Rmd:110]: ah okay, makes sense of course.

**Comment by hadley** [comment]
Did you mean to push changes?

**Comment by ramiromagno** [comment]
yeah :)

**Comment by hadley** [comment]
Thanks!

**Comment by ramiromagno** [comment]
Thank you.

**Pull request merged**